windows/install.cmd

```diff
@@ -0,0 +1,114 @@
+@ECHO OFF
+SETLOCAL
+
+SET MS_EMOJI_FONT_PATH="%SystemRoot%\Fonts\seguiemj.ttf"
+SET MS_FONT_PATH="%SystemRoot%\Fonts\seguisym.ttf"
+SET EMOJI_FONT_PATH="%CD%\EmojiOneColor-SVGinOT.ttf"
+SET FINAL_EMJ_FONT_PATH_NO_QUOTES=%CD%\Segoe UI Emoji with EmojiOne.ttf
+SET FINAL_EMJ_FONT_PATH="%FINAL_EMJ_FONT_PATH_NO_QUOTES%"
+SET FINAL_FONT_PATH_NO_QUOTES=%CD%\Segoe UI Symbol with EmojiOne.ttf
+SET FINAL_FONT_PATH="%FINAL_FONT_PATH_NO_QUOTES%"
+SET CMAP_SCRIPT="%CD%\generate_cmap4_from_12.py"
+
+ECHO Checking if Segoe UI Emoji is installed
+
+REM Windows 8 uses Segoe UI Emoji in addition to Symbol
+REM Windows 7 only uses Segoe UI Symbol
+REM We have to replace _both_ 
+ECHO Checking if Segoe UI Symbol is installed.
+
+IF NOT EXIST %MS_FONT_PATH% (
+    ECHO.
+    ECHO You don't seem to have the Segoe UI Symbol Font installed.
+    ECHO https://support.microsoft.com/en-us/kb/2729094
+    GOTO :ERROR
+)
+
+ECHO Checking if prerequisites are installed.
+
+WHERE python >nul 2>nul || (
+    ECHO.
+    ECHO Python not available in PATH. Is it installed?
+    ECHO https://www.python.org/downloads/windows/
+    GOTO :ERROR
+)
+WHERE pyftmerge >nul 2>nul || (
+    ECHO.
+    ECHO FontTools 3.x is not available in PATH. Is it installed?
+    ECHO https://github.com/behdad/fonttools
+    GOTO :ERROR
+)
+
+PUSHD %TEMP%
+IF EXIST %MS_EMOJI_FONT_PATH% (
+    ECHO Creating new Segoe UI Emoji font from EmojiOne
+    ttx -t "name" -o "emjname.ttx" %MS_EMOJI_FONT_PATH% || GOTO :ERROR
+    ttx -o %FINAL_EMJ_FONT_PATH% -m %EMOJI_FONT_PATH% "emjname.ttx" || GOTO :ERROR
+    DEL "emjname.ttx"
+)
+
+ECHO Creating new Segoe UI Symbol font from EmojiOne
+REM Merge Segoe UI Symbol into EmojiOne, this keeps emoji one's glyph ids intact
+REM for the 'SVG ' table data
+pyftmerge %EMOJI_FONT_PATH% %MS_FONT_PATH%
+REM pyftmerge doesn't generate a cmap4 table if either font has a cmap12 table
```

Comment:
I actually fixed this in [FontTools](https://github.com/behdad/fonttools/pull/635), so it will be in whatever their next release is. The emoji font has the correct tables, it was FontTool's fault that the merge was incorrect. That's what the python script corrects, and what my PR for them fixed.